### ISSUE TITLE
Add Example Usage for public Cloud Run service

### ISSUE THREAD

**Issue by steren** [hashicorp/terraform-provider-google#13417]
it is extremely common to make Cloud Run services public, i.e. to allow any identity to invoke the service (give `allUsers` the IAM invoker role) 

I suggest adding to [the docs](https://registry.terraform.io/providers/hashicorp/google/latest/docs/resources/cloud_run_v2_service) this usage example just below [Example Usage - Cloudrunv2 Service Basic](https://registry.terraform.io/providers/hashicorp/google/latest/docs/resources/cloud_run_v2_service#example-usage---cloudrunv2-service-basic)

```
provider "google" {
    project = "my-project"
}

resource "google_cloud_run_v2_service" "default" {
  name     = "cloudrun-service"
  location = "us-central1"
  ingress = "INGRESS_TRAFFIC_ALL"

  template {
    containers {
      image = "us-docker.pkg.dev/cloudrun/container/hello"
    }
  }
}

 data "google_iam_policy" "noauth" {
   binding {
     role = "roles/run.invoker"
     members = ["allUsers"]
   }
 }

 resource "google_cloud_run_service_iam_policy" "noauth" {
   location    = google_cloud_run_v2_service.default.location
   project     = google_cloud_run_v2_service.default.project
   service     = google_cloud_run_v2_service.default.name

   policy_data = data.google_iam_policy.noauth.policy_data
 }
```

**Comment by cstanger** [comment]
@rileykarson I agree that this is important to add - and this issue is very old. How can i support moving foward? I'm not allowed to give reviews

**Comment by jbbqqf** [comment]
Hi! Triaging older Cloud Run docs requests — I think this one shipped quietly some time ago.

The provider docs for `google_cloud_run_service` already include an **Example Usage - Cloud Run Service Noauth** section that shows the public-invoker pattern with `google_iam_policy` + `allUsers` + `google_cloud_run_service_iam_policy`. See [`website/docs/r/cloud_run_service.html.markdown` lines 139-173](https://github.com/hashicorp/terraform-provider-google/blob/main/website/docs/r/cloud_run_service.html.markdown#L139-L173):

```hcl
data "google_iam_policy" "noauth" {
  binding {
    role = "roles/run.invoker"
    members = [
      "allUsers",
    ]
  }
}

resource "google_cloud_run_service_iam_policy" "noauth" {
  location    = google_cloud_run_service.default.location
  project     = google_cloud_run_service.default.project
  service     = google_cloud_run_service.default.name
  policy_data = data.google_iam_policy.noauth.policy_data
}
```

Rendered: https://registry.terraform.io/providers/hashicorp/google/latest/docs/resources/cloud_run_service#example-usage---cloud-run-service-noauth

If you were specifically after a `google_cloud_run_v2_service` example (the v2 docs don't carry the same Noauth section last I checked), happy to chase that as a separate, smaller doc PR — just say the word. Otherwise, would you mind closing this?

(Disclosure: I drafted this comment with help from Claude Code while triaging stale issues; the docs file was inspected manually on the `terraform-provider-google` `main` branch.)
